credential-free/spring-cloud-azure-starter-jdbc-mysql/spring-cloud-azure-mysql-sample/src/main/resources/schema.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS todo;
+CREATE TABLE todo (id SERIAL PRIMARY KEY, description VARCHAR(255), details VARCHAR(4096), done BOOLEAN);
```

Comment:
When will this SQL been executed? Which artifact will be responsible to execute this SQL? Is it possible to create this table atomically?

Reply:
Yes, as described here:https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto.data-initialization.using-basic-sql-scripts

Reply:
Can we use `spring.jpa.generate-ddl=true` instead of the SQL script here?

Reply:
Could we reuse generate-ddl with Spring Data JDBC?